leaflet-mapbox-gl.js

```diff
@@ -142,9 +145,9 @@ L.MapboxGL = L.Layer.extend({
         if (gl.transform.width !== size.x || gl.transform.height !== size.y) {
             container.style.width  = size.x + 'px';
             container.style.height = size.y + 'px';
-            gl.resize();
+            gl._resize();
```

Comment:
When I resize the window, this line throws the following exception: `Uncaught TypeError: gl._resize is not a function`. Changing it back to `gl.resize()` appears to fix it, but I'm not sure if that caused something else to break. 

Tested using Leaflet 1.0.1 and mapbox-gl-js versions 0.25.1 and 0.26.0.